components/st-microelectronics/landpatterns.stanza

```diff
@@ -181,3 +181,169 @@ public pcb-landpattern WLCSP168:
 
 public pcb-landpattern WLCSP180:
   make-bga-landpattern(14, 13, 0.4, 0.25, tol(5.337, 0.035), tol(6.095, 0.035), PerimeterMatrix(7 through 8, 7 to 8))
+
+;========== New ==========
+
+; public pcb-landpattern SO8N:
+; public pcb-landpattern TSSOP14:
+
+;This land pattern has an error because of zero pins on the east side.
+; public pcb-landpattern TSSOP20:
```

Comment:
TSSOP20 is an n-pin SOIC land pattern. PM me for a doc, it's one of the pending usage examples.